test/repl/proxysql_repl_tests/bin/debezium-repl-config.bash

```
@@ -16,17 +16,8 @@ fi
 echo -n "Configuring connection: mysql1(source) => proxysql => debezium(replication stream) ..."
 
-echo -n "Waiting for kafka_connect to be ready ..."
-kc_ready=0
-for _i in $(seq 1 60); do
-    if curl -s -f http://kafka_connect.${INFRA}:8083/connectors >/dev/null 2>&1; then kc_ready=1; break; fi
-    echo -n "."; sleep 2
-done
-if [ "$kc_ready" -ne 1 ]; then echo " ERROR: kafka_connect not ready after 120s" >&2; exit 1; fi
-echo " ready."
-
 conn_mysql=`envsubst < conf/debezium/register-proxysql.json`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)